nautobot/wireless/factory.py

```diff
@@ -31,7 +31,7 @@ class Meta:
     name = UniqueFaker("company")
     has_description = NautobotBoolIterator()
     description = factory.Maybe("has_description", factory.Faker("sentence"), "")
-    controller = random_instance(Controller, allow_null=True)
+    controller = random_instance(Controller.objects.filter(controller_device__isnull=True), allow_null=True)
```

Comment:
I forget whether this works as written or whether you need to wrap the filtered queryset in a lambda. If it works, it works, just something tickling my mind.

Reply:
I know there are cases lambda was required, but this seemed to work for me.